platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/EditorStyledText.kt

```diff
@@ -34,6 +35,7 @@ fun EditorStyledText(
     resolveMentionDisplay: (text: String, url: String) -> TextDisplay = RichTextEditorDefaults.MentionDisplay,
     resolveRoomMentionDisplay: () -> TextDisplay = RichTextEditorDefaults.RoomMentionDisplay,
     onLinkClickedListener: ((String) -> Unit) = {},
+    onTextLayout: (Layout) -> Unit = {},
```

Comment:
Is there a reason why the parameter is not called `onTextLayoutChanged`?

Reply:
Not really, I think I just missed the mismatch here.